calico-cloud/visibility/elastic/archive-storage.mdx

```diff
@@ -22,6 +22,14 @@ This how-to guide uses the following {{prodname}} features:
 
 - **LogCollector** resource
 
+## Before you begin
+
+**Supported logs for export**
+
+- Syslog - flow, dns, idsevents, audit
+- Amazon S3 - l7, flow, dns, runtime, compliance, audit
```

Comment:
[In the ticket](https://tigera.atlassian.net/browse/DOCS-1418?focusedCommentId=64261) it's mentioned that we will not be adding compliance to this document.

Reply:
As previous comment.